docs: sql references (#205)
GreptimeTeam/docs: docs/reference/sql/limit.md

`LIMIT` clause is used to limit the number of rows returned by a query. This clause is particularly
useful when working with large data sets, as it allows for faster query performance by reducing the
amount of data that needs to be processed.

# Syntax

> The basic syntax of the LIMIT clause is as follows:

Reviewer: ```suggestion
The basic syntax of the `LIMIT` clause is as follows:
```

```sql
SELECT column1, column2, ...
FROM table_name
LIMIT number_of_rows;
```

> The number_of_rows parameter specifies the maximum number of rows to return. If the value of this parameter is

Reviewer: ```suggestion
The number_of_rows parameter specifies the maximum number of rows to be returned. If the value of this parameter is
```

> negative, then no rows are returned.

Reviewer: ```suggestion
negative, no rows will be returned.
```

Consider the following table named "system_metrics":

+-------+-------+----------+-------------+-----------+---------------------+
| host  | idc   | cpu_util | memory_util | disk_util | ts                  |
+-------+-------+----------+-------------+-----------+---------------------+
| host1 | idc_a |     11.8 |        10.3 |      10.3 | 2022-11-03 03:39:57 |
| host1 | idc_b |       50 |        66.7 |      40.6 | 2022-11-03 03:39:57 |
| host1 | idc_c |     50.1 |        66.8 |      40.8 | 2022-11-03 03:39:57 |
| host1 | idc_e |     NULL |        66.7 |      40.6 | 2022-11-03 03:39:57 |
| host2 | idc_a |     80.1 |        70.3 |        90 | 2022-11-03 03:39:57 |
+-------+-------+----------+-------------+-----------+---------------------+

> To retrieve the top 3 rows by memory_util, we can use the LIMIT clause:

Reviewer: ```suggestion
To retrieve the top 3 rows by `memory_util`, we can use the` LIMIT` clause:
```
